rust/types/src/metadata.rs

```diff
@@ -118,7 +119,15 @@ MetadataValue
 */
 
 #[derive(
-    Clone, Debug, Deserialize, PartialEq, PartialOrd, Serialize, FromPyObject, IntoPyObject,
+    Clone,
```

Comment:
We should figure out why your formatter disagrees with everyone else's before merging this.

Reply:
I think the formatter agreed, but this one additional item caused it to wrap.

(Algorithm seems to be `if line_length > x {column} else {inline}